apps/ledger-live-mobile/e2e/specs/speculos/addAccount/AddAccount.ts

```diff
@@ -0,0 +1,35 @@
+import { Application } from "../../../page";
+import { Currency } from "@ledgerhq/live-common/e2e/enum/Currency";
+
+export async function runAddAccountTest(currency: Currency, tmsLink: string) {
+  let app: Application;
+  let deviceNumber: number;
+
+  describe(`Add accounts - ${currency.name}`, () => {
+    beforeAll(async () => {
+      app = await Application.init("onboardingcompleted");
+      await app.portfolio.waitForPortfolioPageToLoad();
+    });
+
+    $TmsLink(tmsLink);
+    it(`Perform an add account`, async () => {
+      await app.addAccount.openViaDeeplink();
+      await app.common.performSearch(currency.name);
+      await app.addAccount.selectCurrency(currency.currencyId);
+
+      deviceNumber = await app.common.addSpeculos(currency.speculosApp.name);
```

Comment:
can we start speculos in before all hook?

Reply:
I am investigating this in my following ticket with the use of CLI.

Reply:
ok, thanks!

Reply:
I did the changes, as adding CLI commands seems more complicated than anticipated